packit/config/common_package_config.py

```diff
@@ -596,3 +596,20 @@ def get_package_names_as_env(self) -> dict[str, str]:
                 return env
             raise PackitConfigException("No packages in config")
         raise PackitConfigException("Multiple packages in config")
+
+    def get_specfile_path_env(self) -> dict[str, str]:
```

Comment:
I just noticed that you've added this to `MultiplePackages` the specfile is generally package specific, but I don't think you have an interface to move it to the `CommonPackageConfig`. For now though, replicate the `if for` logic from `get_package_names_as_env` until we see what should be done there.